web/packages/teleport/src/services/auth/types.ts

```diff
@@ -70,3 +72,17 @@ export type ResetPasswordWithWebauthnReqWithEvent = {
   req: NewCredentialRequest;
   eventMeta?: EventMeta;
 };
+
+export type CreateAuthenticateChallengeRequest = {
+  scope: MfaChallengeScope;
+  allowReuse?: boolean;
+  isMfaRequiredRequest?: IsMfaRequiredRequest;
+  userVerificationRequirement?: UserVerificationRequirement;
```

Comment:
@codingllama If you don't insist, I'd like to keep it as is, since it's idiomatic on the frontend side to have strongly typed fields that lack proper validation on deserialization. And here, in this case, we are talking about a type that won't even be deserialized, just serialized; here, the benefit of specifying type explicitly is having the code self-documented and checked internally.

Reply:
Sure, I'm not really a frontend reviewer, so it's your call.